docs/readmes/resources/ref_magma_metrics.md

```diff
@@ -0,0 +1,186 @@
+---
+id: ref_magma_metrics
+title: Life of a Magma Metric
+hide_title: true
+---
+
+## Life of a Magma Metric
+
+This document will cover the life of a Magma metric as it originates at the
+access gateway (AGW), goes through the orchestrator (orc8r) to the [Prometheus
+Edge Hub](https://github.com/facebookincubator/prometheus-edge-hub), and eventually gets displayed on the NMS UI.
+
+### Phase 1: Collection and Export
```

Comment:
Only "Title Case" the h1 headers, otherwise use "Sentence case"

https://magma.github.io/magma/docs/next/contributing/contribute_conventions#markdown

Reply:
Make this an H2